lib/DDGC/Schema/Result/User.pm

```diff
@@ -4,6 +4,8 @@ package DDGC::Schema::Result::User;
 
 use Moo;
 extends 'DDGC::Schema::Result';
+with 'DDGC::Schema::Role::Result::User::Subscription';
+
```

Comment:
Makes sense, but I think having a two-thousand line role is the same problem as a large result class so plan to not explicitly eschew the idea of composing result classes from multiple roles.